gui/src/pages/Jobs.svelte

```diff
@@ -0,0 +1,57 @@
+<script lang="ts">
+  import Layout from '../components/Layout.svelte';
+  import Link from '../components/Link.svelte';
+  import { api } from '../lib/api';
+
+  const tasks = api.kernel.describeTasks();
+  const jobs = tasks.then((tasks) =>
+    tasks.filter((task) => task.parentId === null),
+  );
+</script>
+
+<Layout>
+  <section>
+    Jobs
+    <!-- TODO: common loader component -->
+    {#await jobs}
+      Loading...
+    {:then jobs}
+      {#if jobs.length === 0}
+        <i>None</i>
+      {:else}
+        <table>
```

Comment:
Suggestion: Wrap in `<CheckeredTableWrapper>` and use `<tbody>` in addition to `<thead>`